### ISSUE TITLE
Min Max value to normalize Abdominal datasets.

### ISSUE THREAD

**Issue by mohammadrezanaderi4** [cchen-cc/SIFA#56]
Hello,
can you please provide the min max value that you used to normalize synapse and chaos dataset to the range of [0, 1].

**Comment by dongdongtong** [comment]
Same question.

I derived the min-max value from the preprocessed abdominal training dataset https://github.com/cchen-cc/SIFA/issues/51#issuecomment-1164099069, and here are my min-max results:
MR: (-1.6470603, 5.4715056)
CT:(-2.0952902, 4.4036083)

Next, I used the derived min-max values and **the adapted abdominal models** downloaded from the [dropbox](https://www.dropbox.com/sh/787kmmuhvh3e3yb/AAC4qxBJTWwQ1UMN5psrN96ja?dl=0) to test on target test cases in https://github.com/cchen-cc/SIFA/issues/51#issuecomment-1164099069 (i.e., CT: 1/9/30/32/33/39, MR: 1/13/32/38). Here is my adaptation performance:
For MR2CT direction, the adapted segmentation Dice is 81.4, while the TMI paper reports **83.4**. The detailed Dice for the four abdominal organs are:
Dice:
Liver:88.0(3.6)
R.kidney:82.5(6.4)
L.kidney:76.6(13.9)
Spleen:78.4(13.0)
Mean:81.4

For CT2MR direction, the adapted segmentation Dice is 85.0, subtlely lower than the paper 85.4. The detailed Dice  are:
Dice:
Liver:89.4(2.6)
R.kidney:89.1(5.9)
L.kidney:79.7(21.2)
Spleen:81.6(10.9)
Mean:85.0
